keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md

```diff
@@ -497,6 +497,13 @@ An architectural change to the current testing policies has been implemented and
 
 *This implementation runs continuously as part of the Antrea CNI project.*
 
+In order to avoid disruptions and minimize risks and/or regressions, new and old implementation will live in parallel during some time:
+
+- The new network policy framework will be added in a new folder `kubernetes/test/e2e/network/netpol`
+- The new network policy framework will use and additional tag [Feature:Netpol]
+
+Once the new implementation has finished and is stable (it should be able to pass 10/10 jobs consecutively without any code change ), the legacy tests and the additional tag will be removed.
```

Comment:
by jobs I mean jobs with the same load as we are running in the CI, running all the suite of tests in parallel, i.e. (`focus=[Conformance]|network` with ginkgo parallel 25), not just the network policy tests. 
Main problems and flakes comes when things interact.
Should I clarify it here or did we understand the same?